lib/edge.js

```diff
@@ -92,11 +92,17 @@ class Edge {
       type: this.type,
       name: this.name,
       spec: this.spec,
+      bundled: this.bundled,
       ...(error ? { error } : {}),
       ...(from ? { from: from.explain(null, seen) } : {}),
     }
   }
 
+  get bundled () {
+    const fromBundleDependencies = this.from && this.from.package && this.from.package.bundleDependencies
+    return (fromBundleDependencies && this.to) ? fromBundleDependencies.includes(this.to.name) : false
+  }
```

Comment:
is this worth memoizing?

Reply:
its currently only used in `explain`

Reply:
`this.from` will always be set unless the edge is in a detached state.

`this.from.package` will always be set (it'll at least be `{}`).

`this.name` is the name of the edge, which will typically always match the name of the node it resolves to.  (A node name doesn't _always_ match its location in `node_modules`, but only because a node might not _be_ in `node_modules`, but in that case, an edge can't resolve to it.)

So, this bit can be simplified to:

```js
get bundled () {
  if (this.error === 'DETACHED')
    return false
  const { bundleDependencies = [] } = this.from
  return bundleDependencies.includes(this.name)
}
```

Reply:
my extra defensive code here is partially dealing with the tests which are incomplete non-normalized pojos

Reply:
> this.name is the name of the edge, which will typically always match the name of the node it resolves to. 

when would it not match the name of the node

Reply:
is it better to check if its in a detached state which has the side effect of not having a `from`, rather than just checking directly for a from?

Reply:
did this, but checked directly for `this.from` instead of the error state